resources/build/build-utils.md

```diff
@@ -144,6 +156,15 @@ another script:
   Options can be used to provide additional data, by including `=<varname>` in
   their definition. Otherwise, they are treated as a boolean.
 
+* **dependencies**: these are other builder scripts which must be configured and
+  built before the actions in this script can continue. Only `configure` and
+  `build` actions are ever passed to dependency scripts; if you are working on
+  code within a dependency, you are currently expected to rebuild and test that
+  dependency locally.
```

Comment:
From the builder.sh meeting, did you mention **dependencies** being located outside the current parent folder (vs child script) ?
Where dependencies typically have `../` in the path

Reply:
```suggestion
  dependency locally.
  
  A dependency is similar to, but not the same as, a child project. Child
  projects live in sub-folders of the parent project, whereas generally a
  dependency will be in another folder altogether. 
```

Reply:
```suggestion
  dependency locally.
  
  A dependency is similar to, but not the same as, a child project. Child
  projects live in sub-folders of the parent project, whereas generally a
  dependency will be in another folder altogether. 
```